.github/workflows/publish_crates.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish Crates
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [master]
```

Comment:
I think it is better if we trigger release on pushing to `stable` branch so PR can trigger which can be done by anyone.